test/e2e-go/features/participation/onlineOfflineParticipation_test.go

```diff
@@ -159,35 +174,45 @@ func TestNewAccountCanGoOnlineAndParticipate(t *testing.T) {
 	newAccountStatus, err := client.AccountInformation(newAccount)
 	a.NoError(err, "client should be able to get information about new account")
 	a.Equal(basics.Online.String(), newAccountStatus.Status, "new account should be online")
+
+	nodeStatus, _ = client.Status()
+	fundedRoundBefore := nodeStatus.LastRound
+	
 	// account receives almost all of rich account's stake (minus enough to
 	// keep it over MinBalance), so it will be selected for participation
 	amountToSend := richBalance - 3*transactionFee - amountToSendInitial - minAcctBalance
 	fixture.SendMoneyAndWait(onlineRound, amountToSend, transactionFee, richAccount, newAccount)
 
 	nodeStatus, _ = client.Status()
-	fundedRound := nodeStatus.LastRound
+	fundedRoundAfter := nodeStatus.LastRound	
 
-	params, err := client.ConsensusParams(nodeStatus.LastRound)
+	params, err := client.ConsensusParams(fundedRoundAfter)
 	a.NoError(err)
-	lookbackRound := balanceRound(basics.Round(nodeStatus.LastRound), params)
-	delta := int64(nodeStatus.LastRound) - int64(lookbackRound)
-	a.True(delta > 0)
+	fundedAccountRoundBefore := balanceRoundOf(basics.Round(fundedRoundBefore), params)
+	fundedAccountRoundAfter := balanceRoundOf(basics.Round(fundedRoundAfter), params)
 
 	// Need to wait for funding to take effect on selection, then we can see if we're participating
 	// Stop before the account should become eligible for selection so we can ensure it wasn't
-	err = fixture.WaitForRoundWithTimeout(fundedRound + uint64(delta) - 1)
+	err = fixture.ClientWaitForRound(fixture.AlgodClient, uint64(fundedAccountRoundBefore),
+		time.Duration(uint64(globals.MaxTimePerRound) * uint64(fundedAccountRoundBefore)))
+	err = fixture.WaitForRoundWithTimeout(uint64(fundedAccountRoundBefore))
 	a.NoError(err)
 
-	blockWasProposed := fixture.VerifyBlockProposed(newAccount, int(delta)-1)
-	a.False(blockWasProposed, "account should not be selected until BalLookback (round %d) passes", int(delta)-1)
+	blockWasProposed := fixture.VerifyBlockProposedRange(newAccount, int(fundedAccountRoundBefore),
+		int(fundedAccountRoundBefore)-1)
+	a.False(blockWasProposed, "account should not be selected until BalLookback (round %d) passes", int(fundedAccountRoundBefore))
 
-	// check that account starts participating after a while
-	proposalWindow := 20 // arbitrary
-	blockWasProposedByNewAccountRecently := waitForAccountToProposeBlock(a, &fixture, newAccount, proposalWindow)
+	// Now wait until the round where the funded account will be used. Give additonal rounds for stability. 
+	err = fixture.ClientWaitForRound(fixture.AlgodClient, uint64(fundedAccountRoundAfter) + 4,
+		time.Duration(uint64(globals.MaxTimePerRound) * uint64(fundedAccountRoundAfter + 4)))
+	a.NoError(err)
```

Comment:
while it's ok to wait for 4 rounds, it should be needed. Assuming the fundedAccountRoundAfter calculation is correct, the proposer for these rounds MUST be the new account, since the old account has no money.